mirage.opam

```diff
@@ -28,6 +28,7 @@ depends: [
   "alcotest" {with-test}
   "fmt" {with-test}
 ]
+available: opam-version >= "2.1.0"
```

Comment:
isn't this (or the functoria one above) sufficient? I'd expect the `-runtime` packages to _not_ require any specific opam version.

Reply:
yeah, I can delete that for `-runtime` package. I did a systemic patch :+1:.